modules/image_and_time.py

```diff
@@ -0,0 +1,40 @@
+"""
+Image and timestamp.
+"""
+import time
+
+import numpy as np
+
+
+# Basically a struct
+# pylint: disable=too-few-public-methods
+class ImageAndTime:
+    """
+    Contains image and timestamp.
+    """
+    __create_key = object()
```

Comment:
Do we want to write a test file for this class?

Reply:
Actually never mind. There really isn't much to test as this class is mostly used for holding information.

Reply:
Yes, but out of scope of this PR.